examples/workflow.py

```diff
@@ -0,0 +1,36 @@
+import os
+from indico import IndicoClient, IndicoConfig
+from indico.queries import ListWorkflowsForDataset, WorkflowSubmission, JobStatus, RetrieveStorageObject
+
+
+host = "app"
+user = "beta"
+domain = "indico.io"
+home = os.environ["HOME"]
+token_path = f"{home}/indico/tokens/{host}-{user}/indico_api_token.txt"
```

Comment:
All other examples in the docs have much simpler token path/ host variables. I think it would be cleaner to continue with the conventional `my_config = IndicoConfig(
    host="app.indico.io", api_token_path="./path/to/indico_api_token.txt"
)`

Reply:
fixed